### PR TITLE
✨Source Lemlist : Migrate Python CDK to Low-code CDK

### LINKED ISSUE #29452: lemlist
[public-apis](https://github.com/airbytehq/airbyte/tree/1219e66fb966f03620db5a2157148e8a5b28abdb/airbyte-integrations/connectors/source-public-apis)

### PR DESCRIPTION
**Pull request by avirajsingh7**
## What
Migrating Source Lemlist to Low-Code CDK
Closes  #29452

## How
Developed using (Configuration Based Source) low-code CDK

## Recommended reading order

1. manifest.yaml
2. schemas/*

## 🚨 User Impact 🚨

*Are there any breaking changes? What is the end result perceived by the user?*

No breaking change

 ##  Test
<details>
<summary>Acceptance Test</summary>
Screenshot 2023-08-14 at 11 18 18 PM

IF THIS TWO TEST ARE IMPORTANT CAN YOU HELP ME HOW CAN I RESOLVE THIS

Screenshot 2023-08-14 at 11 22 55 PM

Screenshot 2023-08-14 at 11 23 11 PM

</details>

<details>
<summary>Migrate Connector : Source Lemlist</summary>

- [x] **Community Member? Grant Edit Access to Maintainers [instructions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests)**
- [x] **Secrets in the Connector's Spec**
- [x] **Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run ./gradlew :airbyte-integrations:connectors:<name>:integrationTest.**
- [x] **Code Reviews Completed**
  - [x] **Documentation Updated**
  - [x] **Connector's README.md**
  - [x] **Connector's bootstrap.md**
  - [x] **docs/integrations/<source or destination>/<name>.md (including changelog. See changelog example)**
  - [x] **docs/integrations/README.md**
  - [x] **airbyte-integrations/builds.md**
- [x] **PR Name follows PR Naming Conventions**
</details>

### CODE REVIEW

**Comment by sh4sh** [comment]
Hey @avirajsingh7 could you share the integration test results?

**Comment by avirajsingh7** [comment]
> Hey @avirajsingh7 could you share the integration test results?

Same here already shared [here](https://user-images.githubusercontent.com/100823015/260530970-79db43a5-c0b2-473b-aa46-d9a6c2176e01.png)

File: airbyte-integrations/connectors/source-lemlist/setup.py [hunk 1]
```diff
@@ -6,10 +6,15 @@
...
-    "airbyte-cdk",
+    "airbyte-cdk~=0.1",
 ]
 
-TEST_REQUIREMENTS = ["requests-mock~=1.9.3", "pytest~=6.2.5", "pytest-mock~=3.6.1", "responses~=0.14.0"]
+TEST_REQUIREMENTS = [
+    "requests-mock~=1.9.3",
+    "pytest~=6.2",
+    "pytest-mock~=3.6.1",
+    "connector-acceptance-test",
```
[the remark this answers is not in the document]
Comment (avirajsingh7) [reply airbyte-integrations/connectors/source-lemlist/setup.py]: @sh4sh  acceptance was not working at that time that's why I have gone with that approach,

File: airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json [hunk 2]
```diff
@@ -18,24 +19,116 @@
...
+      }
+    },
+    "dataIntegrityChecksCron": {
+      "type": "object",
+      "properties": {
+        "errors": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json:91]:
```suggestion
            "type": "object",
            "additionalProperties": true,
            "properties": {
              "text": {
                "type": "string"
              }
            }
```

File: airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json [hunk 3]
```diff
@@ -18,24 +19,120 @@
...
+      "properties": {
+        "errors": {
+          "type": "array",
+            "type": "object",
+            "additionalProperties": true,
+            "properties": {
+              "text": {
+                "type": "string"
+              }
+            }
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json:95]:
```suggestion
        "errors": {
          "type": "array",
          "items": {
            "type": "object",
            "additionalProperties": true,
            "properties": {
              "text": {
                "type": "string"
              }
            }
          }
```

File: airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json [hunk 4]
```diff
@@ -18,24 +19,122 @@
...
+                "format": "%Y-%m-%dT%H:%M:%S.%fZ",
+                "type": ["null", "string"]
+              }
+            }
+          }
+        }
+      }
+    },
+    "revenueVisualization": {
+      "type": "object",
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json:68]:
```suggestion
      "type": "object",
      "additionalProperties": true,
```

File: airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json [hunk 5]
```diff
@@ -18,24 +19,123 @@
...
-          "type": ["null", "boolean"]
+        "averageContractValueCurrency": {
+          "type": "string"
         },
-        "plan": {
+        "conversionRate": {
+          "type": "integer"
+        }
+      }
+    },
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json:84]:
```suggestion
    },
    "_updatedAt": {
      "type": "string"
    },
```

File: airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json [hunk 6]
```diff
@@ -18,24 +19,126 @@
...
         },
-        "ok": {
-          "type": ["null", "boolean"]
+        "conversionRate": {
+          "type": "integer"
+        }
+      }
+    },
+    "_updatedAt": {
+      "type": "string"
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-lemlist/source_lemlist/schemas/team.json:86]:
```suggestion
    "_updatedAt": {
      "format": "%Y-%m-%dT%H:%M:%S.%fZ",
      "type": ["null", "string"]
```

**Review by sh4sh — APPROVED** [review]

File: airbyte-integrations/connectors/source-lemlist/metadata.yaml [hunk 7]
```diff
@@ -1,24 +1,25 @@
 data:
+  allowedHosts:
+    hosts:
+      - "*"
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-lemlist/metadata.yaml:4]:
```suggestion
      - api.lemlist.com
```

**Comment by sh4sh** [comment]
Thank you for contributing!

**Pull request merged**